JSON/Validation.json

```diff
@@ -14,6 +15,6 @@
   "Published": "2020-11-24",
   "SortKey": "VALIDATION",
   "SourceUrl": "https://github.com/RockDood/YesNoNeedy",
-  "SteamID": "2297541858",
```

Comment:
This should be reverted. SteamID is a string due to previous int overflow error.